### PR TITLE
perf: 'text' endpoint default params

### PR DESCRIPTION
**Pull request by crazydevlegend**
- update api integration due to changes with the endpoint
- update default system prompt

### CODE REVIEW

File: utils/server/index.ts [hunk 1]
```diff
@@ -23,7 +23,6 @@ export const BitAPAIConversation = async (
     },
     method: 'POST',
     body: JSON.stringify({
-      return_all: true,
       messages: [
```
Comment (salahawk) [inline utils/server/index.ts:26]: default is:
`count` = 20
`return_all` = false
`exclude_unavailable` = true

Am I right?

File: utils/app/const.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "**[Start New Session]** You are an AI assistant. Follow the user's instructions carefully. Respond using markdown.";
```
Comment (salahawk) [inline utils/app/const.ts:3]: Removing the explanation about bittensor network, miners might give wrong answers just incase they lack knowledge about the network...

File: utils/server/index.ts (hunk 1 above)
Comment (crazydevlegend) [reply utils/server/index.ts:26]: exactly. those are values mog had asked

File: utils/app/const.ts (hunk 2 above)
Comment (crazydevlegend) [reply utils/app/const.ts:3]: it depends on the models the miners run on their servers

**Review by salahawk — APPROVED** [review]

**Pull request merged**